src/core_plugins/kibana/public/home/load_tutorials.js

```diff
@@ -1,3 +1,4 @@
+import 'whatwg-fetch';
```

Comment:
Should this be imported `webpackShims` or someplace higher level so the polyfill is not buried in the home app?

Reply:
If we wanna start using fetch elsewhere, yeah... good point

Reply:
Yeah, it should be somewhere higher. Maybe in the `chrome`? cc @spalger, as we talked about this for other things a while ago

Reply:
yeah, `ui/chrome` is always the first module loaded. it should go at the top of that file

Reply:
right next to `import 'babel-polyfill';` would probably work